### DISCUSSION TITLE
Welcome to teable Discussions!

### DISCUSSION THREAD

**Discussion by tea-artist** [teableio/teable#423] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ctavolazzi** [comment]
Hello! Super excited about this!

I've been interesting in spinning up an Airtable based SaaS but the API rates for testing were out of my budget for the scale I'm working with. 

This is a really great solution. 

Thanks for building it, your work is deeply appreciated.
